server/src/test/java/org/elasticsearch/index/fielddata/BinaryDVFieldDataTests.java

```diff
@@ -118,7 +118,7 @@ public void testDocValue() throws Exception {
 
         // Test whether BinaryDocValuesField makes a deepcopy
         fieldData = indexFieldData.load(reader);
-        BinaryDocValuesField binaryDocValuesField = (BinaryDocValuesField) fieldData.getScriptField("test");
+        BinaryDocValuesField binaryDocValuesField = (BinaryDocValuesField) fieldData.getScriptFieldFactory("test");
```

Comment:
This seems like it shoud be `fieldData.getScriptField("test")`

Reply:
Updated this.